docs/_includes/common/icon.html

```diff
@@ -189,4 +189,11 @@ <h3>Notice Icons</h3>
     </div>
 
     {% include ds{{ page.ds}}/icon-custom.html %}
+
+
+    {% if page.ds == 6 %}
+
+        {% include ds{{ page.ds}}/icon-small.html %}
```

Comment:
I know its more logic, but in the docs I think the small icons should follow after their associated large icons.
Like `Filter Icons` followed by `Small Filter Icons`.  Otherwise users will see the first icon that looks like what they need and never scroll down to the small size.

Reply:
Done!